database/migration/src/main/resources/metadata_changesets/add_metadata_archive_status.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog objectQuotingStrategy="QUOTE_ALL_OBJECTS"
+                   xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd">
+
+    <changeSet id="add_metadata_archive_status" author="cjllanwarne" dbms="hsqldb,mariadb,mysql,postgresql">
+        <addColumn tableName="WORKFLOW_METADATA_SUMMARY_ENTRY">
+            <column name="METADATA_ARCHIVE_STATUS" type="VARCHAR(30)" defaultValue="NULL">
```

Comment:
Take this string `defaultValue` out. We never use `defaultValues`. Instead we always insert the column values from our scala code. (except for db generated primary keys of course).

```xml
<column name="METADATA_ARCHIVE_STATUS" type="VARCHAR(30)">
```

Relatedly, on most DBMS the [original liquibase line](https://www.liquibase.org/documentation/changes/add_default_value.html) is similar to saying in Java

```java
String myNullString = "NULL";
```

It just so happens that some DBMS translate `"NULL" == null` 😞